### PR TITLE
Add a basic logger, helper file for pyenv and placeholder test dir

### PR DESCRIPTION
**Pull request by mooster531**

### CODE REVIEW

File: .python-version [hunk 1]
```diff
@@ -0,0 +1 @@
+3.12
```
Comment (bh2smith) [inline .python-version:1]: Are we already using pyenv?

**Review by bh2smith — COMMENTED** [review]
Are we going to replace all the prints with the logger here?

File: .python-version (hunk 1 above)
Comment (mooster531) [reply .python-version:1]: I am, it's a convenient helper. I was going to open a new PR with more creature comforts for development, but this doesn't really hurt to have already in there IMO. I'll delete it if you're against it.

**Comment by mooster531** [comment]
> Are we going to replace all the prints with the logger here?

I'd say that would be the end goal.

File: .python-version (hunk 1 above)
Comment (bh2smith) [reply .python-version:1]: I wanna use it. Put whenever is meant to go into the project for it. But maybe do a dedicated PR with some instructions for noobs.

**Comment by bh2smith** [comment]
When replace print statements?

**Comment by mooster531** [comment]
> When replace print statements?

Done :)

**Review by bh2smith — APPROVED** [review]

**Pull request merged**